README.md

```diff
@@ -18,7 +18,7 @@ The KES docs use [Hugo](https://www.gohogo.io) to generate static HTML pages.
 
 ## Prerequisites
 
-- Any [operation system that Hugo supports](https://gohugo.io/installation/)
+- Any [operating system that Hugo supports](https://gohugo.io/installation/)
 - [nodejs](https://nodejs.org/en/download/package-manager/) 14.5.0 or later
```

Comment:
Noting here it *seems* like Node 16+ is required for now. Rushan mentioned poking around to restore 14.5.0 support, but absent that Node 16 worked fine and I can still build sphinx docs with it, so no worries there.

Reply:
Updated the version to 16.9.x or later. We can always change that again later if we get a workaround for 14.5.0